elements/pfe-card/src/pfe-card.scss

```diff
@@ -117,7 +111,7 @@ $LOCAL: card;
     }
 
     ::slotted(img) {
-      max-width: 100%;
+      max-width: 100% !important;
```

Comment:
why the importants? was this not working in IE or something?

Reply:
Exactly

Reply:
It also forces the style to prevent bootstrap-type stylesheets from bleeding in and having unintended effects.